frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.styled.tsx

```diff
@@ -61,3 +62,8 @@ export const ImplicitActionMessage = styled.span`
   display: block;
   margin-top: 0.5rem;
 `;
+
+export const InlineSeparator = styled.span`
+  margin-left: ${space(0)};
```

Comment:
IMO, we shouldn't move away from the mockup because the `space` function doesn't have a corresponding value. We don't yet have a strict and consistent spacing guideline, and `space` was introduced as an ad-hoc helper vs. a way to work with well-thought spacing values, so IMO it shouldn't drive how things look

Reply:
that's fair, I'll use the exact number instead.